index.html

```diff
@@ -8,47 +8,40 @@
-  
-              <label for="y-divisions">y div:</label>
-              <output name="y-divisionsOutput" id="y-divisions-output" value="15" class="slider-output"></output>
-              <input type="range" min="5" max="80"  value="15" class="slider" id="y-divisions" data-key="yDivisions">
-  
-              <label for="ratio">ratio:</label>
-              <output name="ratio" id="ratio-output" value="50" class="slider-output"></output>
-              <input type="range" min="0" max="100"  value="50" class="slider" id="ratio" data-key="reductionRatio">
-            </fieldset>
-            <fieldset>
-              <label for="x-offset">x offset:</label>
-              <output name="x-offset" id="x-offset-output" value="1" class="slider-output"></output>
-              <input type="range" min="0" max="100"  value="1" class="slider" id="x-offset" data-key="xOffset">
-  
-              <label for="y-offset">y offset:</label>
-              <output name="y-offset" id="y-offset-output" value="1" class="slider-output"></output>
-              <input type="range" min="0" max="100"  value="1" class="slider" id="y-offset" data-key="yOffset">
-            </fieldset>
+            <label for="x-divisions">x div:</label>
+            <output name="x-divisionsOutput" id="x-divisions-output" value="15" class="slider-output" ></output>
+            <input type="range" min="5" max="100" value="15" class="slider" id="x-divisions" data-key="xDivisions">
+            <label for="y-divisions">y div:</label>
+            <output name="y-divisionsOutput" id="y-divisions-output" value="15" class="slider-output"></output>
+            <input type="range" min="5" max="80"  value="15" class="slider" id="y-divisions" data-key="yDivisions">
+            <label for="ratio">ratio:</label>
+            <output name="ratio" id="ratio-output" value="50" class="slider-output"></output>
+            <input type="range" min="0" max="100"  value="50" class="slider" id="ratio" data-key="reductionRatio">
+            <label for="x-offset">x offset:</label>
+            <output name="x-offset" id="x-offset-output" value="1" class="slider-output"></output>
+            <input type="range" min="0" max="100"  value="1" class="slider" id="x-offset" data-key="xOffset">
+            <label for="y-offset">y offset:</label>
+            <output name="y-offset" id="y-offset-output" value="1" class="slider-output"></output>
+            <input type="range" min="0" max="100"  value="1" class="slider" id="y-offset" data-key="yOffset">
           </div>
         </form>          
       </div>
       <div id="output-display">
         <!--output image will get injected here-->
       </div>
     </section>
-    <section id="footer"><h2>
-        footer
-    </h2></section>
+    <section id="footer">
+      <p>A project by <a href="https://github.com/mrhut10">mrhut10</a> and <a href="https://github.com/khutten">khutten</a>.</p>
```

Comment:
I wonder if we should have a credits page where we have the usernames of anyone who has contributed, could be automated with the github API potentially????
if @khutten @ismaeldcom like this idea i can make a new issue for it, and between us make it a reality.

Reply:
I like the idea, I might make the issue if I get around to it before you.